### ISSUE TITLE
Can I do wildcard domain names and octohost on the same IP address?

### ISSUE THREAD

**Issue by NigelThorne** [octohost/octohost#70]
tl;dr;
If I set up a wildcard dns for my domain (*.nigelthorne.com) to point to 10.10.240.23 (the ipaddress of eth0 on my octohost vm on my local network) ... is it possible to get nginx hosting on port 80 and 443 on eth0, so it proxies to itself on the relevant ports?   ( so upstream was it's own ipaddress) 

Longer:

I have IT hosting my octohost VM.  From my work PC I can see the 10.10.240.x ipaddress, (eth0 on the vm image) so I can ssh to the box and run octo commands and docker commands. 

I can visit all the hosted sites on their docker allocated ports... so 10.10.240.23:45670 or whatever. 

However.. as the VM is not running on my machine, and I'm not on the same subnet as 192.168.x.y (the octodev.io address which is set as a private network for vagrant on eth1) ... I can't see this address or ping it.  Obviously this breaks the wildcard domain name reverse proxy. (which I love)

I was wondering if it's technically possible to only use a single IP address... so have octohost running on 10.10.240.23 (say... set by dhcp server). If I set up a wildcard dns for my domain (*.nigelthorne.com) to point to 10.10.240.23... is it possible to get nginx hosting on port 80 on eth0, so it proxies to itself on the relevant ports.   ( so upstream was it's own ipaddress) 

I only know enough about networking to be dangerous.

Is this possible?  Any pointers? 

Nigel
